nats/README.md

```diff
@@ -1,14 +1,12 @@
-# Nats capability provider for wasmcloud:messaging
+# NATS Capability Provider
+This capability provider is an implementation of the `wasmcloud:messaging` contract. It exposes publish, request, and subscribe functionality to actors.
 
-The nats capability provider exposes publish and subscribe functionality to actors.
-To configure, use the following link settings:
+## Link Definition Configuration Settings
+To configure this provider, use the following link settings in link definitions:
 
-- `SUBSCRIPTION` a comma-separated list of subscription topics. If a subscription is a queue subscription, follow the subscription with `|` and the queue group name. For example, the setting
-    `SUBSCRIPTION=example.actor,example.task|work_queue`
-subscribes to the topic "example.actor" and the topic "example.task" in the queue group "work_queue".
-- `URI` nats connection uri. If not specified, the default is '0.0.0.0:4222'
-- `CLIENT_JWT` jwt auth token. For jwt authentication, both CLIENT_JWT and CLIENT_SEED must be provided.
-- `CLIENT_SEED` private seed for jwt authentication.
-
-For examples of invoking the messaging api, see
-- The example actor [nats-messaging](https://github.com/wasmCloud/examples/tree/main/actor/nats-messaging)
+| Property | Description |
+| :--- | :--- | 
+| `SUBSCRIPTION` | A comma-separated list of subscription topics. If a subscription is a queue subscription, follow the subscription with "\|" and the queue group name. For example, the setting `SUBSCRIPTION=example.actor,example.task\|work_queue` subscribes to the topic `example.actor` and the topic `example.task` in the queue group `work_queue`. |
```

Comment:
in my markdown preview, the \ escape before the bar in double quotes works correctly , so just the bar shows up in preview, but the \ after 'example.task' is visible

Reply:
I like the table format!

Reply:
Odd, in my markdown preview, the `\` does _not_ show up.

Reply:
This is how github renders it:
![image](https://user-images.githubusercontent.com/13748641/131572877-e2546e7a-c37c-4c09-9ef1-5b36c6862443.png)